### ISSUE TITLE
Feature request : sort by name asc/desc

### ISSUE THREAD

**Issue by SaltScratch** [conceptualspace/yet-another-speed-dial#91]
Can we get sort by name (ascending or descending) added to the sort options?

**Comment by klausph** [comment]
Yes, it would very very certain for me, to see bookmarks in YASD sorted by name. mostly ascending. Thanks a lot, Yours klaus

**Comment by kajoken** [comment]
You can already do that in a way. The bookmarks in YASD have the same order as the bookmarks in the created "Speed Dial" Folder. If you sort those by name, the speed dials will also be sorted by name. It's not as comfortable to use but it's possible.

**Comment by michaelpturner** [comment]
> You can already do that in a way. The bookmarks in YASD have the same order as the bookmarks in the created "Speed Dial" Folder. If you sort those by name, the speed dials will also be sorted by name. It's not as comfortable to use but it's possible.

As of 3/24 in Chrome, this is not the case. The Chrome Bookmarks Manager shows them sorted by Name but the actual dials do not sort by name—only be newest/oldest per the addons settings.

**Comment by kajoken** [comment]
> > You can already do that in a way. The bookmarks in YASD have the same order as the bookmarks in the created "Speed Dial" Folder. If you sort those by name, the speed dials will also be sorted by name. It's not as comfortable to use but it's possible.
> 
> As of 3/24 in Chrome, this is not the case. The Chrome Bookmarks Manager shows them sorted by Name but the actual dials do not sort by name—only be newest/oldest per the addons settings.

At least for Chromium I cannot confirm this. I'm on Chromium 123.0.6312.86 right now and it still works exactly the same.
